recipes/geographiclib-cpp/bld.bat

```diff
@@ -0,0 +1,17 @@
+setlocal EnableDelayedExpansion
+
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+    -DGEOGRAPHICLIB_LIB_TYPE:STRING=SHARED ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_BUILD_TYPE:STRING=Release ^
+    ..
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
```

Comment:
Is there a ctest or make test we can run here?

Reply:
Done